packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -299,11 +313,22 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
       uint256 aStableTokenValueInGold = stableTokenSupply.mul(goldAmount).div(stableAmount);
       stableTokensValueInGold = stableTokensValueInGold.add(aStableTokenValueInGold);
     }
+    return
+      FixidityLib
+        .newFixed(reserveGoldBalance)
+        .divide(cgldWeight)
+        .divide(FixidityLib.newFixed(stableTokensValueInGold))
+        .unwrap();
+  }
 
+  /**
+   * @notice Computes a tobin tax based on the reserve ratio.
+   * @return The numerator of the tobin tax amount, where the denominator is 1000.
+   */
+  function computeTobinTax() private view returns (FixidityLib.Fraction memory) {
     // The protocol calls for a 0.5% transfer tax on Celo Gold when the reserve ratio < 2.
-    // The protocol aims to keep half of the reserve value in gold, thus the reserve ratio
-    // is two when the value of gold in the reserve is equal to the total supply of stable tokens.
-    if (reserveGoldBalance >= stableTokensValueInGold) {
+    FixidityLib.Fraction memory ratio = FixidityLib.wrap(getReserveRatio());
+    if (ratio.gte(FixidityLib.newFixed(2))) {
```

Comment:
Could we make the cutoff (currently set to two) a governable parameter that one can query? Could be that we would even want to set this to zero initially and turn the tobin tax on later.

Reply:
Good idea, ticket is here: https://github.com/celo-org/celo-monorepo/issues/2431